src/components/AddItemForm.js

```diff
@@ -1,28 +1,75 @@
 import React from 'react';
-import { collection, addDoc } from 'firebase/firestore';
+import {
+  collection,
+  addDoc,
+  query,
+  where,
+  onSnapshot,
+  getDocs,
+} from 'firebase/firestore';
 import { db } from '../lib/firebase';
 import './AddItemForm.css';
 
 function AddItemForm() {
-  const submitItem = (event) => {
+  const normalizeValue = (value) => {
```

Comment:
Nitpick - This is a good candidate for a helper function. Something that you can move out of this component and into its own file. You would then import it like `import { normalizeValue } from 'helper_file';` where `helper_file` is the file location. 

This provides a few benefits:
- It's easier to test (not that you're writing tests right now).
- It's not coupled to the Add Item Form component. If needed you could use this function elsewhere. 
- It reduces the number of lines in Add Item Form which makes it a little more readable.